From: Michael James Gratton <mike@vee.net>
Date: Wed, 15 Nov 2017 14:42:54 +1100
Subject: Fix a GLib critical warning when opening a file chooser.

Opening a file chooser is currently emitting:

> GLib-CRITICAL **: '&ay' is not a valid GVariant format string

This fixes that.

* src/filechooser.c (handle_open): Correct the GVariant format for
  bytestrings.

Origin: upstream, 0.10, commit:1ae4f920ec9b360741142c817a5a6934359c5f2a
---
 src/filechooser.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/filechooser.c b/src/filechooser.c
index 141dd14..420354b 100644
--- a/src/filechooser.c
+++ b/src/filechooser.c
@@ -454,9 +454,9 @@ handle_open (XdpImplFileChooser *object,
        * In a sandboxed situation, the current folder and current file
        * are likely in the fuse filesystem
        */
-      if (g_variant_lookup (arg_options, "current_folder", "&ay", &path))
+      if (g_variant_lookup (arg_options, "current_folder", "^&ay", &path))
         gtk_file_chooser_set_current_folder (GTK_FILE_CHOOSER (dialog), path);
-      if (g_variant_lookup (arg_options, "current_file", "&ay", &path))
+      if (g_variant_lookup (arg_options, "current_file", "^&ay", &path))
         gtk_file_chooser_select_filename (GTK_FILE_CHOOSER (dialog), path);
     }
 
